src/pages/taesooKim/Main/Feed.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import CommentList from "./CommentList";
+import BtnLike from "./BtnLike";
+import Like from "./Like";
+
+export class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      value: "",
+      comment: [],
+      isClicked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ comment: this.props.commentList });
+  }
+
+  getValue = e => {
+    this.setState({
+      value: e.target.value,
+    });
+  };
+
+  handleAdd = e => {
+    if (this.state.value) {
+      e.preventDefault();
+      const newComment = {
+        id: Date.now(),
+        userName: "orosy.ts",
+        content: this.state.value,
+        isUser: true,
+      };
+      const comment = [...this.state.comment, newComment];
+      this.setState({ comment, value: "" });
+    }
+    e.preventDefault();
+  };
```

Comment:
```suggestion
  handleAdd = e => {
    const { value, comment: prevComments } = this.state
    e.preventDefault();
    
    if (!value) return
    
    const newComment = {
      id: Date.now(),
      userName: "orosy.ts",
      content: value,
      isUser: true,
    };
    
    const comments = [...prevComments, newComment];
    
    this.setState({ comments, value: "" });
  };
```
early return을 적절히 활용하면 중괄호의 중첩을 줄일 수 있습니다

Reply:
네네 수정하였습니다🐸